### PR TITLE
feat(combobox-multi-select): DLT-2066 chip truncation

### PR DESCRIPTION
**Pull request by braddialpad**
# feat(combobox-multi-select): DLT-2066 chip truncation

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

- [x] Feature

## :book: Jira Ticket

https://dialpad.atlassian.net/browse/DLT-2066

## :book: Description

Added new prop chipMaxWidth that will truncate text if the chip is greater than that width.

## :bulb: Context

Some users are having issues with multiselect in smaller width interfaces.

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.
- [x] I have added all relevant documentation.
- [x] I have considered the performance impact of my change.

## :camera: Screenshots / GIFs

Screenshot 2024-09-19 at 11 32 52 AM

### CODE REVIEW

File: packages/dialtone-vue3/recipes/comboboxes/combobox_multi_select/combobox_multi_select.vue [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<!-- eslint-disable vue/no-static-inline-styles -->
```
Comment (juliodialpad) [inline packages/dialtone-vue3/recipes/comboboxes/combobox_multi_select/combobox_multi_select.vue:1]: why is this line needed? The same applies for Vue 2 version

**Review by juliodialpad — APPROVED** [review]
This works pretty well, along with the change I made recently, should be enough for the users to avoid overflowing issues with this component.

Not related to this change but the focus ring looks pretty strange.

Screenshot 2024-09-19 at 1 49 03 p m

File: packages/dialtone-vue3/recipes/comboboxes/combobox_multi_select/combobox_multi_select.vue (hunk 1 above)
Comment (braddialpad) [reply packages/dialtone-vue3/recipes/comboboxes/combobox_multi_select/combobox_multi_select.vue:1]: because we're setting `:style` directly on the component which is generally not something we want to do, but in this case it makes sense because the style is passed in via a prop and we have to set it dynamically.

**Comment by braddialpad** [comment]
> Not related to this change but the focus ring looks pretty strange.

hmm this IS actually related to this change. it's fine on https://dialtone.dialpad.com/vue

**Comment by braddialpad** [comment]
Problem fixed, I removed the overflow: hidden from the truncation class. Doesn't seem to hurt the truncation at all.
